Microsoft.Toolkit.Graph.Controls/Controls/GraphPresenter/QueryOption.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Graph.Controls
+{
+    /// <summary>
+    /// XAML Proxy for <see cref="Microsoft.Graph.QueryOption"/>.
+    /// </summary>
+    public sealed class QueryOption
```

Comment:
Shouldn't we reuse the existing ListDictionary and keep from adding another Key<->Value mapping class?
Something like this:
```xaml
...
xmlns:specialized="using:System.Collections.Specialized"
...
<specialized:ListDictionary>
    <x:String x:Key="key1">Value1</x:String>
    <x:String x:Key="key2">Value2</x:String>
</specialized:ListDictionary>
```
One less class to maintain.

Reply:
I didn't know this existed 🙂. I do like having close mapping to the intent though for the Graph. And we'd lose the implicit operator converter, eh? Let's chat quick in the morning.

Reply:
I don't know how we bind to Value in these cases, so let's leave this for now and explore more later. I think there does need to be some refactoring to ensure timings and/or dynamic updates to query options...